### PR TITLE
add more information for guest transcript

### PR DESCRIPTION
**Pull request by ArabellaJi**
This PR is to enrich the guest transcript. Besides, when I'm working on this, I find that the "Experience Transcript" button disappeared. Perhaps someone deleted it accidentally, so I added it back.

### CODE REVIEW

**Comment by ArabellaJi** [comment]
The Experience Transcript looks like this now:
image

**Review by jsenning — APPROVED** [review]
This looks good and displays nicely.

File: src/views/CoCurricularTranscript/index.tsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const CoCurricularTranscript = () => {
         setLoading(true);
         const transcriptItems = await transcriptService.getItems(profile.AD_Username);
         setTranscriptItems(transcriptItems);
+        console.log(transcriptItems);
```
[the remark this answers is not in the document]
Comment (ArabellaJi) [reply src/views/CoCurricularTranscript/index.tsx:37]: Thank you for noticing this! It's for testing and I'll remove it now

**Review by jsenning — APPROVED** [review]
Looks good to me

**Pull request merged**